conf/modules.config

```diff
@@ -22,6 +22,31 @@ process {
         ext.args = '--quiet'
     }
 
+    withName: 'KRAKEN2_KRAKEN2' {
+        publishDir = [
+            path: { "${params.outdir}/kraken2_reports" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: 'KRONA_KTIMPORTTAXONOMY' {
+        publishDir = [
+            path: { "${params.outdir}/kraken2_reports/krona_reports" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: 'UNTAR' {
+        publishDir = [
```

Comment:
I'm not sure you want to output the kraken db, because it's size can be huge (depending on the selected one) and also it has been previously downloaded by the user, so already in user's device? You may want to use the storeDir in case you want to store the db and reuse it for later without the need of publishing it in the output

Reply:
I see what you mean! I could create a patch to the UNTAR module to add the storeDir directive, that would also need some changes to the config but it can be done.

In any case, to avoid unnecessary waste of space by saving the uncompressed database, the pipeline works differently if the user provides a gzipped database or an uncompressed one. If the pipeline is gzipped, the `UNTAR` module uncompresses it and uses it, but by default, it won't save the uncompressed database if the user provided a compressed database.

The outputting of the uncompressed kraken2 db is turned off by default by the `params.save_uncompressed_k2db`, which is set as `false`. On the `modules.config` file this is read by the `enable` declaration.

If the database is uncompressed, and the user passes a path to the `kraken2_db` param, the `UNTAR` module is not called; the database is simply used and remains in the user's original directory.